src/Console/Arguments.php

```diff
@@ -129,24 +129,44 @@ public function getArgument(string $name): ?string
     /**
      * Get an array of all the options
      *
-     * @return array<string, string|bool|null>
+     * @return array<string, list<string>|string|bool|null>
      */
     public function getOptions(): array
     {
         return $this->options;
     }
 
     /**
-     * Get an option's value or null
+     * Get a non-multiple option's value or null if not set.
      *
      * @param string $name The name of the option to check.
-     * @return string|bool|null The option value or null.
+     * @return string|bool|null
      */
     public function getOption(string $name): string|bool|null
     {
         return $this->options[$name] ?? null;
     }
 
+    /**
+     * Get a boolean option's value or null if not set.
+     *
+     * @return bool|null
+     */
+    public function getBooleanOption(string $name): ?bool
+    {
+        return $this->options[$name] ?? null;
```

Comment:
Should we have a cast/assertion so that psalm can figure out what is going on?

Reply:
I don't think a cast is correct here as the value should be bool or the user shouldn't use this method.

Reply:
Thats what the assert asserts though